crates/uv-client/src/base_client.rs

```diff
@@ -350,16 +350,6 @@ enum Security {
 }
 
 impl BaseClient {
-    /// The underlying [`ClientWithMiddleware`] for secure requests.
-    pub fn client(&self) -> ClientWithMiddleware {
-        self.client.clone()
-    }
-
-    /// The underlying [`Client`] without middleware.
-    pub fn raw_client(&self) -> Client {
-        self.raw_client.clone()
-    }
```

Comment:
Is this going to break a downstream consumer?

Reply:
Who would be affected? I can't find anything

```
$ git log -S "fn raw_client"
commit e7ae0f50d218aede8d7283d8cf414e8f7782fbba (origin/main, origin/HEAD, main)
Author: konsti <konstin@mailbox.org>
Date:   Tue Oct 22 13:36:18 2024 +0200

    Respect allow insecure host in publish (#8440)

commit 205bf8cabef5c931203df15bca5e524dbfaac08a
Author: konsti <konstin@mailbox.org>
Date:   Tue Sep 24 18:07:20 2024 +0200

    Implement trusted publishing (#7548)
    
    Co-authored-by: Charlie Marsh <charlie.r.marsh@gmail.com>
```